README.md

```diff
@@ -66,21 +65,3 @@ val backend: StorageBackend[Task] = ???
 
 val client = StorageClient(backend)
 ```
-
-### Using strict types (Future etc.)
-
-The library supports only `F[_]: Effect` by default which makes some common (strict) types like `scala.concurrent.Future` impossible to use.  
-There exists a workaround however:
-
-```scala
-import com.avast.clients.storage.{StorageBackend, StorageClient}
-import monix.eval.Task
-import monix.execution.Scheduler
-
-implicit val scheduler: Scheduler = ???
-implicit val taskToFuture: Task ~> Future = ???
-
-val backend: StorageBackend[Task] = ???
-
-val client: StorageClient[Future] = StorageClient[Task](backend).mapK[Future]
-```
```

Comment:
Did you really remove the support for this? If so, why there's still `fkTaskToFuture` present in tests?

Reply:
Originally I've just removed the documentation part, but after your comment, I've completely removed the Future support from the library.